packs/data/equipment.db/reinforced-wheels.json

```diff
@@ -0,0 +1,57 @@
+{
+    "_id": "z48TnIQIcyKfyDpL",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
+    "name": "Reinforced Wheels",
+    "system": {
+        "baseItem": null,
+        "containerId": null,
+        "description": {
+            "value": "<p>The chair's wheels have been reinforced with tough, metal rims or a metal cap. While in the chair, you gain a wheel Strike. Your wheel deals 1d4 bludgeoning damage and has the agile, attached, and free-hand traits. A reinforced wheel is a simple melee weapon in the club weapon group. You can etch weapon runes onto reinforced wheels. A wheelchair can only have one attached weapon.</p>"
```

Comment:
This should be made as a weapon, not equipment, but we'll have to make a new base weapon for it.